src/ResourceManagement/NotificationHubs/Microsoft.Azure.Management.NotificationHubs/generated.cmd

```diff
@@ -0,0 +1,17 @@
+::
+:: Microsoft Azure SDK for Net - Generate library code
+:: Copyright (C) Microsoft Corporation. All Rights Reserved.
+::
+
+@echo off
+set autoRestVersion=0.14.0-Nightly20160125
```

Comment:
you should use 0.16.0-Nightly20160426 of autorest generator in generate.cmd script. That is almost the latest one.